bin/all_sky_search/pycbc_fit_sngls_by_template

```diff
@@ -21,12 +21,13 @@ import pycbc
 from pycbc import events, init_logging
 from pycbc.events import triggers, trigger_fits as trstats
 from pycbc.events import stat as statsmod
+from pycbc.events import cuts
 from pycbc.types.optparse import MultiDetOptionAction
 from pycbc.io import HFile
 
 #### DEFINITIONS AND FUNCTIONS ####
 
-def get_stat(statname, trigs, threshold):
+def get_stat(args, trigs, threshold):
```

Comment:
This looks good at first glance

I'm actually surprised that this line wasn't `args` before, as `statname` doesn't seem to be used at all.

Reply:
Looking at the code history, statname is a legacy from an old code used by single stat calculation. Then the single stat was much simplified and statname was left useless.